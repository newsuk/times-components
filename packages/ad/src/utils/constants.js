export const SHOWCASE_ARTICLE_BASE_URL = "https://www.thetimes.co.uk";
export const AD_COMPOSER_DEFAULT_AD_UNIT = "d.thetimes.co.uk";
export const AD_DEFAULT_BASE_URL = "https://www.thetimes.co.uk";