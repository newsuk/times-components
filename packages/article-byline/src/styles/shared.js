import styleguide from "@times-components/styleguide";

const { colours, fontFactory } = styleguide();
const shared = {
  link: {
    color: colours.functional.action,
    ...fontFactory({
      font: "supporting",
      fontSize: "cardMeta"
    })
  },
  text: {
    ...fontFactory({
      font: "supporting",
      fontSize: "cardMeta"
    }),
    color: colours.functional.secondary,
    flexDirection: "row"
  }
};

export default shared;
