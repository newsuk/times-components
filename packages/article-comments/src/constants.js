export const COMMUNITY_GUIDELINES_BASE_URL = "https://www.thetimes.co.uk";
export const STORE_URL_GB = "https://thetimes.co.uk/subscribe/digital";
export const STORE_URL_IE = "https://store.thetimes.ie/";
export const STORE_URL_GLOBAL = "https://globalstore.thetimes.co.uk/";
export const ARTICLE_BASE_URL = "https://www.thetimes.co.uk";
