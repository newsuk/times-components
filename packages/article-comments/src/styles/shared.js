import styleguide from "@times-components/styleguide";

const { colours, fontFactory, spacing } = styleguide();
const styles = {
  button: {
    marginBottom: spacing(0),
    marginTop: spacing(5),
    maxWidth: 215
  },
  container: {
    alignItems: "center",
    borderStyle: "solid",
    borderTopColor: colours.functional.keyline,
    borderTopWidth: 1,
    marginBottom: spacing(5),
    width: "100%"
  },
  headline: {
    color: colours.functional.primary,
    ...fontFactory({
      font: "headline",
      fontSize: "commentsHeadline"
    }),
    maxWidth: 315,
    paddingBottom: spacing(2),
    paddingTop: spacing(6),
    textAlign: "center"
  },
  link: {
    color: colours.functional.action
  },
  supporting: {
    ...fontFactory({
      font: "supporting",
      fontSize: "tertiary"
    }),
    color: colours.functional.secondary,
    maxWidth: 280,
    textAlign: "center"
  }
};

export default styles;
