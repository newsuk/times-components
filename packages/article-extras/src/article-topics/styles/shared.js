import { colours, tabletWidth } from "@times-components/styleguide";

const sharedStyles = {
  topicsContainer: {
    borderTopColor: colours.functional.keyline,
    borderTopWidth: 1
  },
  topicsContainerTablet: {
    alignSelf: "center",
    width: tabletWidth
  },
  topicsMetaContainer: {
    justifyContent: "flex-start"
  }
};

export default sharedStyles;
