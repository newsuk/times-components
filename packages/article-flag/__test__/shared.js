/* eslint-env jest */

import "react-native";
import React from "react";
// eslint-disable-next-line import/no-extraneous-dependencies
import renderer from "react-test-renderer";

import ArticleFlag, {
  NewArticleFlag,
  UpdatedArticleFlag,
  ExclusiveArticleFlag,
  SponsoredArticleFlag
} from "../article-flag";

module.exports = () => {
  it("renders correctly", () => {
    const tree = renderer.create(<ArticleFlag title="articleFlag" />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it("renders New flag correctly", () => {
    const tree = renderer.create(<NewArticleFlag />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it("renders Updated flag correctly", () => {
    const tree = renderer.create(<UpdatedArticleFlag />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it("renders Exclusive flag correctly", () => {
    const tree = renderer.create(<ExclusiveArticleFlag />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it("renders Sponsored flag correctly", () => {
    const tree = renderer.create(<SponsoredArticleFlag />).toJSON();
    expect(tree).toMatchSnapshot();
  });
};
