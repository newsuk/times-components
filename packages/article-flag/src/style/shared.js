import styleguide from "@times-components/styleguide";

const { fontFactory } = styleguide();
const styles = {
  diamond: {
    marginRight: 5
  },
  title: {
    ...fontFactory({
      font: "bodyRegularSmallCaps",
      fontSize: "cardMetaMobile"
    }),
    fontWeight: "400",
    letterSpacing: 1.4
  },
  view: {
    alignItems: "center",
    flexDirection: "row"
  }
};

export default styles;
