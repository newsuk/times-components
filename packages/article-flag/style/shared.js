import { fonts } from "@times-components/styleguide";

const styles = {
  view: {
    flexDirection: "row",
    alignItems: "center"
  },
  diamond: {
    marginRight: 5,
    marginBottom: 1
  },
  title: {
    fontFamily: fonts.bodyRegularSmallCaps,
    fontSize: 12,
    fontWeight: "400",
    letterSpacing: 1.4
  }
};

export default styles;
