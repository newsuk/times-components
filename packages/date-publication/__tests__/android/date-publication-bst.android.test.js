import shared from "../shared-london";
import { dateBST } from "../constants";

shared(dateBST);
