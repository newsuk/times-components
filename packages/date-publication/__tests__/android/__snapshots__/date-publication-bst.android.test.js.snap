// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`1. date and times publication 1`] = `"Saturday July 01 2017, 5:32am, The Times"`;

exports[`1. date and times publication 2`] = `"Saturday July 01 2017, 3:32pm, The Times"`;

exports[`2. date and sundaytimes publication 1`] = `"Saturday July 01 2017, 5:32am, The Sunday Times"`;

exports[`2. date and sundaytimes publication 2`] = `"Saturday July 01 2017, 3:32pm, The Sunday Times"`;

exports[`3. date and no given publication 1`] = `"Saturday July 01 2017, 5:32am"`;

exports[`3. date and no given publication 2`] = `"Saturday July 01 2017, 3:32pm"`;

exports[`4. date and no day 1`] = `"July 01 2017, 5:32am"`;

exports[`4. date and no day 2`] = `"July 01 2017, 3:32pm"`;
